### PR TITLE
Jordan/after magic link

### LINKED ISSUE #4525: Cleaning up notifications
**now**
- [x] user menu ghost icon is not centred on windows
- [x] "Add an address" should go in user menu
- [x] `CardSignIn` should be modal instead of in page component
- [x] paywall variables and scroll bugs
- [x] message after sign in toggles states should be a modal that displays which addresses you'll receive notifications about -> link and CTA should go to notifications page
- [x] voting power changes message is broken (should show %)
- [x] remove new validator notification for now (and use network icon instead of lunie logo - for future use)
- [x] lunie theme on notifications page instead of network theme
- [x] hide app menu - add logo - deselect current network when on notifications page
- [x] notifications becomes home page / landing page if user is signed in with email
- [x] magic link copy needs to be updated
- [x] click the notifications icon from the paywall and the app gets stuck on a loading screen
- [x]  take user out from notifications back to Paywall when they sign out being on the notifications page

**next**
---
- [ ] connect user emails via intercom
- [ ] drip campaign 
- [ ] only display notifications that go back 30 days
- [ ] every user should have 1 notification from Lunie that says welcome

### PR DESCRIPTION
**Pull request by jbibla**
Closes #4525

**Description:**
- email auth is now a modal with updated copy
- visibility hidden won't break styling for close button on modals
- added centered prop for buttons
- config needed to be imported in account.js

Screen Shot 2020-07-28 at 10 47 57 PM

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f20e2f5e687401435f0c643--lunieio.netlify.app

File: app/src/components/account/EmailAuthentication.vue [hunk 1]
```diff
@@ -1,42 +1,46 @@
...
   },
   computed: {
     ...mapState([`account`]),
   },
   mounted() {
     this.$store.dispatch(`signInUser`, window.location.href)
   },
+  methods: {
+    goToNotifications() {
+      this.$router.push({ name: "notifications" }).catch(err => {
```
Comment (jbibla) [inline app/src/components/account/EmailAuthentication.vue:41]: apparently we should be `.catch`-ing all `router.push`'s now https://github.com/vuejs/vue-router/issues/2873

Comment (faboweb) [reply app/src/components/account/EmailAuthentication.vue:41]: Do we use next(false)?

Comment (Bitcoinera) [reply app/src/components/account/EmailAuthentication.vue:41]: well, that explains all the uncaught Promises appearing on the console then

**Comment by faboweb** [comment]
Looks great btw

File: app/src/components/account/EmailAuthentication.vue (hunk 1 above)
Comment (faboweb) [reply app/src/components/account/EmailAuthentication.vue:41]: I guess this is the issue that we see in Lunie: https://github.com/vuejs/vue-router/issues/2873#issuecomment-549409726

File: app/tests/unit/specs/components/account/__snapshots__/EmailAuthentication.spec.js.snap [hunk 2]
```diff
@@ -1,22 +1,37 @@
...
       
-      You are now signed in! 🚀
+      You're now signed in to Lunie with your email address. Head over to the
+      notifications page to see some of your recent events.
     
-    </h1>
+    </p>
+     
+    <tmbtn-stub
+      centered="true"
```
Comment (faboweb) [inline app/tests/unit/specs/components/account/__snapshots__/EmailAuthentication.spec.js.snap:29]: why do we need to center here and in other places not?

Comment (jbibla) [reply app/tests/unit/specs/components/account/__snapshots__/EmailAuthentication.spec.js.snap:29]: mostly about visual symmetry. when a modal has a form or some inputs the button can go on the right to imply progress and also to balance the weight of the elements visually. 

for a simple message like this one, with one call to action and centered title and text - i think the centered button is the strongest layout.

File: app/src/components/common/TmBtn.vue [hunk 3]
```diff
@@ -141,6 +146,11 @@ export default {
   background: var(--app-nav-hover);
 }
 
+.button.centered {
+  text-align: center;
```
Comment (jbibla) [inline app/src/components/common/TmBtn.vue:150]:
```suggestion
```

**Comment by Bitcoinera** [comment]
This one seems to be failing? The email link is taking me to `CardSignInRequired`, but actually I'm already signed in in cosmos-hub-testnet

image

**Comment by jbibla** [comment]
i didn't change any of the routing logic so i just visited the route directly

**Pull request merged**